odoo/addons/base/models/res_lang.py

```diff
@@ -17,6 +19,40 @@
 DEFAULT_TIME_FORMAT = '%H:%M:%S'
 
 
+class LangData:
```

Comment:
Is this not a `frozendict`?

I don't really see the point of the custom `__bool__`, `__eq__` and `__hash__`
Then the `__repr__` is also useless

Reply:
I build a class so that the LangInfo can be used like a record for cached fields
but you are rigth, inheriting from `frozendict` can make it simpler